src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -827,14 +889,15 @@ export class ProcessedFilesStackedTable extends React.PureComponent {
     }
 
     render(){
-        const { files, collapseLongLists } = this.props;
+        const { files, columnHeaders: propColHeaders, showNotesColumns, collapseLongLists } = this.props;
```

Comment:
Codacy found an issue: ['columnHeaders' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5860998)